src/vue/common/EmailGetter.spec.js

```diff
@@ -13,6 +13,7 @@ Vue.config.silent = true
 
 const localVue = createLocalVue()
 
+// TODO: tests
```

Comment:
resolve this too please

Reply:
Should I run tests or shuld I look some action at site ?

Reply:
remove the comment so